go_test.mod

```diff
@@ -8,15 +8,20 @@ require (
 	github.com/nats-io/nats-server/v2 v2.10.11
 	github.com/nats-io/nkeys v0.4.7
 	github.com/nats-io/nuid v1.0.1
+	github.com/stretchr/testify v1.8.0
```

Comment:
🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺

Reply:
🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺

So much work without it for such minor thing

🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺 🥺